apps/EchoServerClient/DEPS

```diff
@@ -0,0 +1,2 @@
+../../lib/util/*.v3
+../../lib/net/*/*.v3
```

Comment:
Probably want `lib/net/x86-64-linux`; otherwise this will break when new platforms are added.